### DISCUSSION TITLE
Allow overriding "Automatically delete head branches" in PRs

### DISCUSSION THREAD

**Discussion by marcinbak** [community/community#21360] [Pull Requests]
It would be very helpful if PR summary had a _delete head branch after merge_ checkbox that would be checked if _Automatically delete head branches_ is set.
This would allow disabling the automated deletion when not desired and still have a default behavior defined in settings.

**Comment by empro-danielkress** [comment]
This would be very neat. When working with feature branches and auto-deploy I want to clean up a feature branch once done by default. But still be able to integrate a working interim set to main and not have the branch deleted (which in my case would trigger a time consuming deletion & rebuild of IaC controlled resources for the feature branch)
